src/main/java/com/crud/tasks/service/SimpleEmailService.java

```diff
@@ -0,0 +1,43 @@
+package com.crud.tasks.service;
+
+import com.crud.tasks.domain.Mail;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.MailException;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+
+import org.springframework.stereotype.Service;
+
+
+
+@Service
+public class SimpleEmailService {
+    private static final Logger LOGGER = LoggerFactory.getLogger(SimpleMailMessage.class);
+
+    @Autowired
+    private JavaMailSender javaMailSender;
+
+    public void send (final Mail mail) {
+        LOGGER.info("Starting email preparation...");
+        try {
+            javaMailSender.send(createMailMessage(mail));
+            LOGGER.info("Email has been sent.");
+        } catch (MailException e) {
+            LOGGER.error("Failed to process email sending: ", e.getMessage(), e);
+        }
+    }
+
+
+
+    private SimpleMailMessage createMailMessage(final Mail mail) {
+        SimpleMailMessage mailMessage = new SimpleMailMessage();
+        mailMessage.setTo(mail.getMailTo());
+        mailMessage.setSubject(mail.getSubject());
+        mailMessage.setText(mail.getMessage());
+        if((mail.getToCc()!= null) && (!mail.getToCc().equals("")))
+                mailMessage.setCc(mail.getToCc());
```

Comment:
zawsze uzywaj klamer `{ } `. Java pozwala na nie pisanie ich, ale to bardzo w zlym tonie i narzedzia do statycznej analizy kodu (np. sonarqube) wyrzuca Ci to jako blad